src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -1629,6 +1633,7 @@ blockCommands = M.fromList
    , ("lstinputlisting", inputListing)
    , ("inputminted", inputMinted)
    , ("graphicspath", graphicsPath)
+   , ("svgpath", graphicsPath)
```

Comment:
If I understand correctly, then this would also modify the paths searched by `\includegraphics`, which is probably harmless but potentially undesired.